docs/install-docker-quickstart.md

```diff
@@ -1,264 +1,83 @@
-	# Let's pull and migrate the database
-	docker run --link=db heroiclabs/nakama migrate up --database.address root@db:26257
-	# start Nakama server
-	docker run --link=db -p 7350:7350 -p 7351:7351 heroiclabs/nakama --database.address root@db:26257
-	```
-
-Connect to the database SQL shell using the following command:
-
-=== "Shell"
-	```shell
-	docker exec -it "db" /cockroach/cockroach sql --insecure -d nakama
-	```
-
-You can also change Nakama config options simply by editing the last line. For instance:
+1. Pull the required Docker images. From your terminal, run the following:
 
+```sh
+# Pull the latest CockroachDB image
+docker pull cockroachdb/cockroach
+# Pull the latest Nakama image
+docker pull heroiclabs/nakama
 ```
-docker run --link=db -p 7350:7350 -p 7351:7351 heroiclabs/nakama --database.address root@db:26257 --config /path/to/config.yml --socket.server_key "mynewkey"
-```
-
-## Running Nakama with docker-compose
-
-Docker Compose simplifies running more than one Docker container in conjunction. For Nakama, we’ll need two containers: one for Nakama itself and one for the database it relies on, CockroachDB.
-
-You can choose to configure the Nakama and CockroachDB containers without Docker Compose but we do not recommend it when you’re starting out.
-
-Docker Compose uses YAML configuration files to declare which containers to use and how they should work together.
-
-1\. Let’s start by creating the Nakama Docker-Compose file:
 
-Create a file called `docker-compose.yml` and edit it in your favourite text editor:
+When complete, your terminal output should look like this:
 
-=== "docker-compose.yml"
-	```yaml
-	version: '3'
-	services:
-	  cockroachdb:
-            container_name: cockroachdb
-            image: cockroachdb/cockroach:latest-v20.2
-            command: start-single-node --insecure --store=attrs=ssd,path=/var/lib/cockroach/
-            restart: "no"
-            volumes:
-              - data:/var/lib/cockroach
-            expose:
-              - "8080"
-              - "26257"
-            ports:
-              - "26257:26257"
-              - "8080:8080"
-	  nakama:
-	    container_name: nakama
-	    image: heroiclabs/nakama:latest
-	    entrypoint:
-	      - "/bin/sh"
-	      - "-ecx"
-	      - >
-	          /nakama/nakama migrate up --database.address root@cockroachdb:26257 &&
-	          exec /nakama/nakama --name nakama1 --database.address root@cockroachdb:26257
-	    restart: always
-	    links:
-	      - "cockroachdb:db"
-	    depends_on:
-	      - cockroachdb
-	    volumes:
-	      - ./:/nakama/data
-	    expose:
-	      - "7349"
-	      - "7350"
-	      - "7351"
-	    ports:
-	      - "7349:7349"
-	      - "7350:7350"
-	      - "7351:7351"
-	    healthcheck:
-	      test: ["CMD", "curl", "-f", "http://localhost:7350/"]
-	      interval: 10s
-	      timeout: 5s
-	      retries: 5
-	volumes:
-	  data:
-	```
+![Pull docker images](images/docker-pull-result.png)
```

Comment:
nit: about image naming - do you think it's better to put the images used in this document in its own folder (that is named the same as the file name)?

Reply:
Agreed - pretty trivial now but definitely better as the amount of content grows. Done 👍